Content.Shared/Xenoarchaeology/Artifact/XAE/XAEPhasingSystem.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Shared.Xenoarchaeology.Artifact.XAE.Components;
+using Robust.Shared.Physics;
+using Robust.Shared.Physics.Systems;
+
+namespace Content.Shared.Xenoarchaeology.Artifact.XAE;
+
+public sealed class XAEPhasingSystem : BaseXAESystem<XAEPhasingComponent>
```

Comment:
This sounds weird... "XenoarchPhaseThroughWallsSystem"?

Reply:
it phases through artifact container too 🙈

Reply:
Can it be called 'NoCollision' ?